code/hsec-tools/hsec-tools.cabal

```diff
@@ -57,6 +60,10 @@ library
                       -Wincomplete-uni-patterns
                       -Wpartial-fields
                       -Wredundant-constraints
+  if impl(ghc >= 9.2.1)
+    build-depends:    lucid2 >= 0.0.20220508
+  else
+    build-depends:    lucid >= 2.0.0
```

Comment:
Could we pick a single lucid package? It's a bit confusing that this depends on the ghc version.